toolkit/scripts/toolchain/container/jdk8-build-raw.sh

```diff
@@ -118,34 +118,34 @@ popd
 rm -rf libxcb-1.12
 
 # freetype 2
```

Comment:
Are these build instructions different from the ones in spec files? How?

Reply:
In some cases yes, these are simplified build instructions used for bootstrapping the toolchain